### PR TITLE
Paulsbranch

### PR DESCRIPTION
**Pull request by pmcdonald1012**

### CODE REVIEW

File: components/StudentPage/index.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ import React, { useState, useContext, useEffect } from "react";
 import SPTasks from "./SP-Tasks";
 import SPETStag from "./SP-ETStag";
 import SPDependents from "./SP-Dependents";
-import styles from "../../styles/StudentPage.module.css";
-import Chat from "../Chat";
+import styles from "../../styles/StudentPage.module.css"
+// import "../../StyleSheets/StudentPage.css";
+// import SideNav from "../SideNav/SideNav";
+// import LoginContext from "../../Context/LoginContext";
+// import ChatModal from "../../Components/Chat/ChatModal";
```
[the remark this answers is not in the document]
Comment (pmcdonald1012) [reply components/StudentPage/index.js:9]: Not my component

File: components/StudentPage/index.js [hunk 2]
```diff
@@ -2,8 +2,11 @@ import React, { useState, useContext, useEffect } from "react";
 import SPTasks from "./SP-Tasks";
 import SPETStag from "./SP-ETStag";
 import SPDependents from "./SP-Dependents";
-import styles from "../../styles/StudentPage.module.css";
-import Chat from "../Chat";
```
Comment (moses369) [inline components/StudentPage/index.js:6]: this line should be in this file

Comment (pmcdonald1012) [reply components/StudentPage/index.js:6]: My dude, its not my component

Comment (moses369) [reply components/StudentPage/index.js:6]: its in there in main  but not in your branch for some reason

File: components/StudentPage/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (moses369) [reply components/StudentPage/index.js:9]: if we merge this it will just pollute the main branch again

[the remark this answers is not in the document]
Comment (pmcdonald1012) [reply components/StudentPage/index.js:9]: Im just going to replace the file with whats in the main branch

**Pull request closed without merging**